src/main/modules/SnippetsManager.js

```diff
@@ -188,12 +188,31 @@ class SnippetsManager {
     }
 
     replace(value) {
-        const clipboardContent = this.clipboard.readText()
+        try {
+            // 1. store old clipboard data
+            const clipboardContent = clipboardy.readSync()
+            // 2. place value in clipboard
+            clipboardy.writeSync(value)
+            // 3. perform paste (takes 1-3 ms)
+            this.keyboardSimulator.keyTap('v', 'command')
+            // 4. place stored data back in clipboard
+            setTimeout(() => clipboardy.writeSync(clipboardContent), 50)
+            return true
```

Comment:
Returning a boolean may be a nice thing to do, but we are not using this returned information nowhere, so I don't see any value.